### PR TITLE
Add address tokens balance

### PR DESCRIPTION
**Pull request by kolya182**

### CODE REVIEW

File: src/common/TokenParser.ts [hunk 1]
```diff
@@ -151,20 +152,33 @@ export class TokenParser {
...
+                    }
                 });
-                return Promise.all(balancePromises);
             });
         });
     }
 
+    private getTokenBalance(address: string, contractAddress: string) {
+        const tokenAddress: string = address.substring(2);
+        const contractData: string = `0x70a08231000000000000000000000000${tokenAddress}`;
```
Comment (vikmeup) [inline src/common/TokenParser.ts]: What is this `contractData`?

File: src/common/TokenParser.ts [hunk 2]
```diff
@@ -183,7 +197,6 @@ export class TokenParser {
             promises.push(ERC20Contract.findById(tokenContract).then((token: any) => {
                 return {
                     address: token.address,
-                    decimals: token.decimals
```
Comment (vikmeup) [inline src/common/TokenParser.ts]: Why remove `decimals`?

File: src/common/TokenParser.ts (hunk 1 above)
Comment (kolya182) [reply src/common/TokenParser.ts]: Combination of contract 'balanceOf()' ERC20 token function in hex and tokenAddress, required to call the balance of an address.

Comment (kolya182) [reply src/common/TokenParser.ts]: Fixed in next PR

File: src/common/TokenParser.ts (hunk 2 above)
Comment (kolya182) [reply src/common/TokenParser.ts]: It was not mentioned in original task, but added in next PR as well as `symbol`, `name`

**Pull request merged**